### PR TITLE
chore: created a database

### PR DESCRIPTION
**Pull request by yrd369**
For this project we need a database to cross check the condition. I created a Database for this project to validate.

### CODE REVIEW

File: assets/main.js [hunk 1]
```diff
@@ -0,0 +1,355 @@
+// Creating Database
+const database = [
+  {
+    id: 1,
+    acNum: 1029384756,
```
Comment (mshajid) [inline assets/main.js:5]: @yrd369 : try to give a specific name. `acNum` won't considered as a property, What you've to do is try add a specific name like `accountNum`

File: assets/main.js [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+    holderName: "Jack",
+    holderAddress: "5396 North Reese Avenue, Fresno CA 93722",
+    units: "121 kWh"
+  },
+  {
+    id: 2,
+    acNum: 1029384757,
+    holderName: "Jil",
+    holderAddress: "2685 California Street, Mountain View CA 94040",
+    units: "170 kWh"
```
Comment (mshajid) [inline assets/main.js:15]: @yrd369 : **Suggestion** - Why can't we change this to a number also we've to change in the UI also? Let'see how it goes.

**Review by mshajid — COMMENTED** [review]
@yrd369 : I have seen some improvements needs to be done with in the array of objects.

File: assets/main.js (hunk 2 above)
Comment (yrd369) [reply assets/main.js:15]: @mshajid : I have changed the key name as you requested.

File: assets/main.js (hunk 1 above)
Comment (yrd369) [reply assets/main.js]: @mshajid : I have changed the key name as you requested.

**Pull request merged**